README.md

```diff
@@ -7,3 +7,44 @@ Hypha Organizational Handbook (H₂0) describes the vision, processes, and cultu
 *Hypha Worker Co-operative* cultivates collective growth and meaningful livelihoods through learning and building technologies together.
 
 Working in solidarity with our neighbours, we support each other and provide safe harbour to imagine and create alternative futures.
+
+---
+
+## Technologies Used
```

Comment:
Mind if I add some more of my go-to header emojis? :)

```suggestion
## :hammer_and_wrench: Technologies Used
```

Reply:
Or rather, this:

```suggestion
## 🛠Technologies Used
```

The suggestion as a unicode character means the the hash deeplink won't get broken if the emoji is changed later :)

Also, curious your thoughts on removing the space between emoji and first letter? It's a little more "cramped", but means that the deeplink will always be the same, with or without an emoji prefix :)

Reply:
Gist with an example of what I mean:
https://gist.github.com/patcon/03d225a8f365e013d98ad5c39ad28ebc#%EF%B8%8Fheader-with-unicode-emoji-and-no-space

Reply:
Ugh. Actually, just realized that the above link only _looks_ like it doesn't make a diff when viewing in browser bar.

![Screen Shot 2019-10-28 at 11 41 24 AM](https://user-images.githubusercontent.com/305339/67713210-260f4b00-f9a4-11e9-994c-0d5e3e5d9b31.png)

But if I actually type that out as https://gist.github.com/patcon/03d225a8f365e013d98ad5c39ad28ebc#header-with-unicode-emoji-and-no-space, the deeplink doesn't work :/

Seems like an attack vector to fuck with browser address bar that way, but def means my "space or not" comment is moot. Probably means better to have space so as to draw explicit attention to it as different, as otherwise it could be confusing that it looks the same in address bar, but there's a hidden emoji character that's being hidden.

Reply:
Final suggestion:

```suggestion
## 🛠 Technologies Used
```